eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         displayName: "Publish artifacts"
         inputs:
           artifactName: packages
-          targetPath: $(Build.ArtifactStagingDirectory)
+          path: $(Build.ArtifactStagingDirectory)
```

Comment:
Was this change intentional?  Per the docs the parameter is named `targetPath`:

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact?view=azure-devops#yaml-snippet

Reply:
Yes https://github.com/Azure/azure-sdk-for-js/pull/4339#discussion_r304614724